bin/submitControlWorkflow.sh

```diff
@@ -165,13 +165,16 @@ if [ $? -ne 0 ]; then
     echo -e "mkdir -p nextflow/${controlJobName}"
     mkdir -p "nextflow/${controlJobName}" && pushd "nextflow/${controlJobName}" > /dev/null
     rm -rf run.out run.err .nextflow.log*  
-    bsub \
-        -J "${controlJobName}" \
-        -M 16000 -R "rusage[mem=16000]" \
-        -u $SCXA_REPORT_EMAIL \
-        -o run.out \
-        -e run.err \
-        "$nextflowCommand" 
+
+    sbatch \
```

Comment:
Needs `--time`, otherwise the sbatch command will fail.

Reply:
done here [46aaf0a](https://github.com/ebi-gene-expression-group/scxa-control-workflow/pull/45/commits/46aaf0a8b912ed34aa454f4bfe7825d2322800c6)